assets/js/base/context/cart-checkout/checkout-state/constants.js

```diff
@@ -26,6 +26,7 @@ export const DEFAULT_STATE = {
 	hasError: false,
 	calculatingCount: 0,
 	orderId: checkoutData.order_id,
+	orderNotes: '',
```

Comment:
I added the `orderNotes` value in the Checkout context. It doesn't belong to the Shipping data neither the Billing data, so adding it to those contexts felt wrong. At the same time, creating a specific context to only hold the order notes seemed a bit overwhelming. That's why I decided to use the Checkout context to store it, but I'm not 100% sure about it either.

Reply:
This is consistent with the approach I took for `createAccount` - makes sense to me.

https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2851#issuecomment-657941513

Reply:
I think checkoutState is fine here.